bugsnag/src/main/java/com/bugsnag/Configuration.java

```diff
@@ -36,9 +36,9 @@ public class Configuration {
     public Delivery delivery;
     public EndpointConfiguration endpointConfiguration;
     public Delivery sessionDelivery;
-    public String[] redactedKeys = new String[]{"password", "secret", "Authorization", "Cookie"};
+    public String[] redactedKeys = new String[] {"password", "secret", "Authorization", "Cookie"};
     public String[] ignoreClasses;
-    public String[] notifyReleaseStages = null;
+    public String[] enabledReleaseStages = null;
```

Comment:
Is it worth changing the type from `String[]` to `Set<String>` at the same time?